### ISSUE TITLE
Add page to tutorial to educate new users on what type of photo is needed

### ISSUE THREAD

**Issue by misaochan** [commons-app/apps-android-commons#95]
From https://meta.wikimedia.org/wiki/Grants_talk:IEG/Improve_%27Upload_to_Commons%27_Android_App 

> I'm concerned by these two lines:
> 
> publicity is needed to inform non-Wikimedians about this app
> account creation from within the app
> The WMF already built a Mobile Media Uploads app aimed at non-editors. It had to be shut down. Editors know about copyright and they upload pictures that are useful for the project. Non-editors didn't. They uploaded massive numbers of selfies and copyvios that had to be researched and deleted. Any images that weren't outright deleted were generally useless. The rate of useful images was so low that it wasn't worth the cost of dredging thought the bad images to find anything worthwhile. Alsee (talk) 13:59, 15 April 2016 (UTC)

I think we could add a page to the tutorial (that new users see after installing the app) about this? To inform new users that selfies and copyright violations will be deleted.

**Comment by nicolas-raoul** [comment]
Very good idea, some education in this field would indeed improve the usefulness of the picture, and the tutorial sounds like a good place for that :-)

**Comment by misaochan** [comment]
Hi @nicolas-raoul , could you comment on this thread: https://commons.wikimedia.org/wiki/Commons:Village_pump#Mobile_edits_and_uploads please? :) I have responded but am unsure if my assumption is correct.

Thanks!

**Comment by nicolas-raoul** [comment]
Note:
Cleaning 5% of https://commons.wikimedia.org/w/index.php?title=Category:Uploaded_with_Mobile/Android just took me an hour.
So, I believe that the problem of useless pictures, while existing, is very far from negating the benefits of the app.

**Comment by misaochan** [comment]
During my chat with Marti today, it was suggested that we pay particular emphasis to this feature, due to concerns about new users uploading selfies and copyvios. She suggested that we contact User:Pine to collaborate on making an effective tutorial. I'll try to talk to Pine and see if he's willing to help.

**Comment by misaochan** [comment]
It was also suggested that we add a measure of success that tests the effectiveness of the tutorial in educating new users who have never contributed to Wikimedia before. Preferably a survey of sorts. But I'm still thinking on how we could get the relevant users to take them. I suppose we could contact them on-wiki after browsing the users who submitted recently uploaded images?

**Comment by nicolas-raoul** [comment]
How about checking the last pictures uploaded by 100 different new users, and classifying them into useful/useless?

By the way, I recently surveyed hundreds of app-uploaded pictures and nominated the few ones I judged useless, but other Commons users found them *not* useless, I was surprised: https://commons.wikimedia.org/wiki/Commons:Deletion_requests/mobile_tracking/archive/2016-21#File:ASCII_image_.png https://commons.wikimedia.org/wiki/Commons:Deletion_requests/mobile_tracking/archive/2016-21#File:Green_insect.jpeg

**Comment by misaochan** [comment]
Hmm, that sounds like a good idea, but how do we know that they are new to Wikimedia without contacting them?

Also I'm not sure if we can get 100 brand new users, haha, but we could try with a smaller number.

**Comment by nicolas-raoul** [comment]
Yes, a few might have created a new account, but maybe that's in an acceptable error margin?
Yes we can check all new users in a week/month instead :-)

**Comment by misaochan** [comment]
Ah, yes, good point. Is there an easy way to tell which accounts are new, or will we need to rely on WMF for that information?

**Comment by nicolas-raoul** [comment]
How about checking the user's Special:Log ?
https://commons.wikimedia.org/wiki/Special:Log/Xaviant_LLC
https://commons.wikimedia.org/wiki/Special:Log/Deshpremiraj

**Comment by misaochan** [comment]
Looks good, thanks!

**Comment by misaochan** [comment]
Hm, I just thought of this... how do we know if the picture is a copyright violation or not? I guess if the Exif data shows that it is taken by a phone camera, it's likely to not be a copyvio?

**Comment by nicolas-raoul** [comment]
On Commons people usually consider that pictures with EXIF are not copyvio.
Most copyvios are images downloaded from websites, and most pictures on websites have no EXIF.

**Comment by misaochan** [comment]
Reminder to self: Add page about licenses (as mentioned at #140) to tutorial.

**Comment by misaochan** [comment]
As suggested by Marti, I've been [talking to Pine ](https://meta.wikimedia.org/wiki/User_talk:Pine#Request_for_collaboration) about collaborating for this tutorial. He has given me a link to his [script outline](https://meta.wikimedia.org/wiki/Grants:IEG/Motivational_and_educational_video_to_introduce_Wikimedia/Scripts) so far, could anyone take a look and see if the style would suit our app tutorial? I don't think we can do a video on the in-app tutorial page, but maybe we could summarize there and maybe have his upcoming Commons video up on the Google Play page instead?

**Comment by nicolas-raoul** [comment]
Do you mean this script?
https://meta.wikimedia.org/wiki/Grants:IEG/Motivational_and_educational_video_to_introduce_Wikimedia/Scripts#Wikimedia_Commons_.286_minutes.29_.28priority_1.29

I see:
> Intended audience: Education instructors, GLAM+STEM project leaders, students, participants in GLAM+STEM activities, and media uploaders such as participants in Wiki Loves Monuments

I think our audience is more "*any random person who installed the Wikipedia app and wondered what that other app in the same developer account was for*" :-)

The irrelevant "Government works" and "OTRS" parts could be edited out I guess.
The rest of the topics sound very relevant.

Using such a video as the Google Play page video is a great idea!

**Comment by misaochan** [comment]
I think those are other modules that he's working on, we will only need to use his upcoming Commons module (that he says he will be starting work on soon). 

Great, I'll contact him again later and see how things go. :)

**Comment by misaochan** [comment]
I just noticed this cartoon: https://commons.wikimedia.org/wiki/Special:UploadWizard

It seems pretty useful. Think we could reuse it in our tutorial?

**Comment by nicolas-raoul** [comment]
It looks like I disabled that cartoon somehow haha... do you have a link to
the image itself? Thanks!

**Comment by misaochan** [comment]
Yeah, I edited it to show the image link instead. :)

**Comment by nicolas-raoul** [comment]
Better have the text as strings so that it can be internationalized, but the general flow and individual cartoons are worth reusing, yes :-)

**Comment by misaochan** [comment]
Update: Pine has linked me his Commons tutorial script, with permission to share - https://docs.google.com/document/d/16UeyGdEySXe3ImUzPCEkileA45aXpXYH30Dfatjvi5I/edit . Any thoughts on how we could use the information there in our tutorial?

I've also been pondering how to structure the tutorial. Currently most of the space is taken up by icons - which are nice and eye-catching, but which leaves little room for text. If we want to make a more in-depth tutorial, we would need to put more content in. Should we cut out the icons on the current tutorial to make more space, so we can have more content without increasing the number of pages the user has to scroll through at the start? Or is adding a few more pages OK?

**Comment by nicolas-raoul** [comment]
We should first extract from that script the passages that apply to the Commons app.
Images are good if they make the reading more pleasant, I am not against adding more pages, like 10 in total would still be OK I guess, if each is very easily understandable.

**Comment by misaochan** [comment]
I agree, I'll extract out the bits that I find relevant and post them here for comment.

If we added up to 10 pages, should we provide the option to skip at the start, for old users who are just updating/reinstalling/etc the app? 

And if we went the images route, we would need to create some of our own. I could pick out open source icons but it would likely turn out the same as what happened when I did that for NearbyList - they won't be very congruent in style, and probably not very nice. :) Shall we ask for volunteers in the village pump again, instead?

**Comment by nicolas-raoul** [comment]
Yes, hopefully someone will try as it is a finite set of like 10 images,
whereas for Nearby the list is never-ending.

**Comment by nicolas-raoul** [comment]
Of course, let's first decide the content before asking for illustrations
:-)

**Comment by misaochan** [comment]
Will do, thanks @nicolas-raoul !

**Comment by misaochan** [comment]
I've extracted passages from the script that I think are relevant, and grouped them. Note that this is still in raw form, I definitely would reword them and summarize them into bullet points once we decide what information we want and what we don't want. :) Please let me know what you think!

**Intro**

Wikimedia Commons hosts most of the images that are used on the Wikimedia projects, including Wikipedia and Wikidata. Commons hosts images that anyone can use without paying a copyright fee.

**What can I upload?**

Examples include natural objects like flowers, useful objects like bicycles, people who are in public spaces, maps, and diagrams.

We can usually upload images that we have created entirely by ourselves. Keep in mind that when we upload an image, we grant everyone permission to use, copy, modify, and sell it. Wikipedia is free of charge, and images on Commons are free of copyright fees. However, we usually keep an important right called attribution, which means that anyone who uses the image must give us a written credit.

We can’t accept works created or inspired by others. You can upload your photographs of old art, statues, and buildings (usually over 150 years old). 

Be respectful of privacy and personality rights. Privacy rights may include the right of a person not to be photographed when they are inside a private place like their home unless the person gives permission. Personality rights may include the right to be shown honestly. Editing a photo could be dishonest if the edits make important changes to the content, such as changing the appearance of someone’s clothing from green to purple.

Some areas may have special restrictions on photography. In many countries, private property owners and government officials may place restrictions on taking photos in places like courtrooms, military facilities, and museums.

**How should I upload it?**

We should give the file a meaningful name that will make it easy for other people to find. The file name can be in any language, e.g. “1965 Ford truck”. For the description, type  “1965 Ford truck in a repair shop”.


Now we need to select categories for the photo. [insert automatic category suggestions example here]

[Manual categories] For instance, type “Ford truck” into the category field. Matching options appear, including “Ford trucks”. VISUAL: select “Ford trucks”.

Being more specific is better when that is possible. Let’s search by location [include example]

Editing the file name, adding descriptions and categories, and adding location information can take a little time, but these actions will make the image much easier for other people to find.

**Comment by nicolas-raoul** [comment]
Sounds good! :-)

Maybe one or two slides could give several real-world examples of good and
bad pictures, a bit like the ✔✗ at the bottom of
http://commons-app.github.io ?

Cheers!
Nicolas

**Comment by misaochan** [comment]
I started working on a draft for the exact pages: https://github.com/commons-app/apps-android-commons/wiki/Tutorial-for-new-users

Thoughts/comments? Is that ready for illustration requests?

@nicolas-raoul I thought I saw the real-world examples on the website before, but can't seem to find them now. Do you have any examples that I could use?

**Comment by nicolas-raoul** [comment]
I don't remember eanything except the "Photo Guideline" part and the list of images that I suggested to use as screenshots.

How about inverting page 5 and 6? To leave people on a positive note :-)

**Comment by misaochan** [comment]
Good idea, did that. :)

Hm, I will try to see if I can find the list again.

**Comment by misaochan** [comment]
Posted on Commons village pump: https://commons.wikimedia.org/wiki/Commons:Village_pump#Wanted:_illustrations_.26_comments_on_tutorial_for_new_users

Hopefully some people volunteer for the illustrations!

**Comment by nicolas-raoul** [comment]
I took the liberty to modify a few things:
- Dumbed down English so that more people understand
- Maps and diagrams are typically not made on mobile
- "Pictures of people"could incitate selfies and taking pictures of
strangers, which is not a prime purpose
https://github.com/commons-app/apps-android-commons/wiki/Tutorial-for-new-users/_compare/9349a5a674328cb29b1c9f2f50310515c5369d26...bee30264e197feb13c9b0156eef7596e59a06be9

I am not sure about
"Areas with special restrictions on photography (e.g. military facilities,
some courtrooms & museums, etc)"
Did you find a policy page about this?
Commons has to obey copyright law but does not care about rules decided by
a museum for instance:
https://commons.wikimedia.org/wiki/Category:No_photography_signs

**Comment by misaochan** [comment]
The modifications look good to me, thanks. :)

I took the 'areas with special restrictions' part directly from Pine's script. Should we ask him for the source?

**Comment by nicolas-raoul** [comment]
Yes thanks :-)

**Comment by misaochan** [comment]
Hm, I looked at the source again and it just says:

> 
> 
> VISUAL: Fade in a bullet point line that says “Some areas may have special restrictions on photography.” (without the external quotes)
> 
> 
> NARRATORS
>  In many countries, private property owners and government officials may place restrictions on taking photos in places like courtrooms, military facilities, and museums.
> 

So I guess I might have misinterpreted it - he is just informing users that they could get in trouble if they take photos there, it might not be actual Commons policy. 

Shall we leave it out in that case? I don't think it's worth spending a lot of text to explain the distinction to users.

**Comment by nicolas-raoul** [comment]
Yes, the video might afford to give general life advice to the
photographers (similar to "don't get too close when taking a picture of an
alligator"), but as for us, I think we should really focus on what is OK/KO
from Commons' point of view only.

**Comment by nicolas-raoul** [comment]
Do pages 5 and 6 need illustrations? In my understanding, it could be a
grid of 2*2 pictures with a short title under each such as "Selfie"
"Downloaded image" etc

**Comment by misaochan** [comment]
I don't think pages 5 and 6 need illustrations (that's why in my village pump post I only asked for pages 1-4 :)).

I'm not sure what sort of layout we should do for pages 5-6 though. I think 2*2 might be a bit too much for smaller phones? I was just intending to have 2 pics per page.

**Comment by misaochan** [comment]
Alternatively, if no illustrations are forthcoming, perhaps we could merge the 'real examples' page with the 'please upload' page, and vice versa for 'please do not upload'... so those could be the 'illustrations' for Page 2 and 3. That has the upside of reducing the # of pages that people need to scroll through each time they update/reinstall the app. 

And I could put in the Ford truck pic for the 'filling in image details' page (Page 4). Not sure what to put on Page 1 though, maybe we could reuse the original illustration.

**Comment by misaochan** [comment]
For the real-life good pics examples, where should we do the attribution? Would be too clunky to do it in the tutorial itself I think.

And for the real-life bad pics, if I put a copyrighted pic as an example, wouldn't I be infringing copyright? :/ For selfies I could just search for open-source ones, I guess.

**Comment by nicolas-raoul** [comment]
Why not! If the pictures are very clear to understand (clearly a selfie,
for instance), they can be the size of a thumbnail.

I think that we should find a better example than "1965 Ford truck" because:
- Prefixing a vehicle is a very American thing to do, but grammatically not
obvious non-natives. I for one am not even sure now whether that's the name
of a car model or just the year it was produced.
- "Truck" is ambiguous, it can mean lorry but in some communities it means
a personal car with a pickup shape.
- Rather not use a brand name.
The difficulty is to find something with a descriptive enough name, yet
understandable by people of very different backgrounds.
How about about something like "Oxford train station"?

About copyright: If we find public domain images (or take them ourselves
and license them as public domain or even GNU-GPL), then no need for
attribution. I can try to take them, just let me know.

**Comment by misaochan** [comment]
Good point, we should probably change the '1965 Ford Truck'. I'll look into some alternatives.

For the good pics examples, I was actually just planning to use examples directly from Commons (such as https://commons.wikimedia.org/wiki/File:Rajasthan-Chittore_Garh_15.jpg and https://commons.wikimedia.org/wiki/File:Kofi_Annan_2012.jpg which was in Pine's script). Do we not need to do attribution if we reuse Commons images?

**Comment by nicolas-raoul** [comment]
I think we do need to attribute, yes.

Here are many public domain pictures:
https://commons.wikimedia.org/wiki/Category:Public_domain

**Comment by misaochan** [comment]
I'm not sure if I'm looking at the wrong subcategories, but pretty much all the images I've seen in that category so far have either been very old pictures (black and white) or diagrams/text/drawings?

If you could take the pics, that would be great. :)

**Comment by misaochan** [comment]
Hi @nicolas-raoul , any luck with the photos? Alternatively I could try using my own, but I am not a very good photographer. :)

**Comment by nicolas-raoul** [comment]
wow sorry I totally forgot about this!
I will try to do it this weekend.

**Comment by misaochan** [comment]
No worries, this weekend is good. Thanks!

**Comment by misaochan** [comment]
Thanks for the pics. Do you think it is a good idea to place a transparent red 'X' over the top of the 'bad pics' to make it easier for lazy readers to understand?

**Comment by nicolas-raoul** [comment]
Yes, good idea!
Some people do not read instructions but are receptive to red Xs.

**Comment by misaochan** [comment]
Hi @nicolas-raoul ,

I've gotten all the pages up except the 'don't upload' page... I'm having issues overlaying the red X over the 'bad' pics. I'm following Inkscape tutorials like this one http://goinkscape.com/how-to-make-inkscapes-background-transparent/ , and then exporting the supposedly transparent file as PNG and trying to copy the X over the selfie etc pics, but the transparency just doesn't show up. This is a screenshot of what I mean, lol.

![transparency issue](https://cloud.githubusercontent.com/assets/3611199/20876143/bfea01b4-bb24-11e6-8be9-7315f23b22a7.png)

Do you know how we could get this to work?

**Comment by nicolas-raoul** [comment]
Select by color zone before making transparent :-)
Sorry no pc for a while

**Comment by misaochan** [comment]
I couldn't really figure it out, so instead I used the 'draw' tool within paint.net itself (it had an inbuilt 'X' symbol option, I didn't know that!). Sorry I wasn't able to use your images.

Should be mostly done soon, I made some small modifications that weren't copied to the wiki, but no major content changes. Will do a release shortly.

**Comment by misaochan** [comment]
Screenshots have been added to the [wiki](https://github.com/commons-app/apps-android-commons/wiki/Tutorial-for-new-users), if no issues will release soon. :)

**Comment by misaochan** [comment]
Released in v1.40. Please do let me know if there are any problems.
